### PR TITLE
[Customer Center] Adds the Customer Center to PaywallsTester

### PR DESCRIPTION
**Pull request by JayShortway**
As the title says. There's a new section called "Customer Center".

### CODE REVIEW

File: Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/SamplePaywallsList.swift [hunk 1]
```diff
@@ -84,6 +84,8 @@ struct SamplePaywallsList: View {
                     introEligibility: Self.introEligibility
                 )
             )
+        case .customerCenter:
+            CustomerCenterView()
```
Comment (tonidero) [inline Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/SamplePaywallsList.swift:88]: I believe this is only available in iOS, so we need to wrap all this code into `#if os(iOS)` macros. We can probably add support for other systems soon, but it's a limitation for now.

File: Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/SamplePaywallsList.swift [hunk 2]
```diff
@@ -136,6 +138,12 @@ struct SamplePaywallsList: View {
                 } label: {
                     TemplateLabel(name: "Unrecognized paywall", icon: "exclamationmark.triangle")
                 }
+                
+                Button {
+                    self.display = .customerCenter
+                } label: {
+                    TemplateLabel(name: "Customer Center", icon: "person.fill.questionmark")
```
Comment (tonidero) [inline Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/SamplePaywallsList.swift:145]: I was actually adding a similar button in my [PR](https://github.com/RevenueCat/purchases-ios/pull/4053). I'm going to update the label to say `Customer Center (sheet)`, so it's clearer. Should we change the label here to `Customer Center (SwiftUI)` so it's more distinct? Maybe it's ok to leave this as is and just change it in other buttons.

**Review by tonidero — APPROVED** [review]
A couple comments but LGTM!

File: Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/SamplePaywallsList.swift (hunk 2 above)
Comment (JayShortway) [reply Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/SamplePaywallsList.swift:145]: Ah good call! Will update the label as you suggested. Should we align on the icon as well?

Comment (tonidero) [reply Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/SamplePaywallsList.swift:145]: I feel we should create another section if we're going to have multiple buttons. As for the icon, we got a close one already :P. I think it's best for it to be slightly different... But honestly doesn't matter too much IMO.

Comment (JayShortway) [reply Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/SamplePaywallsList.swift:145]: A section sounds good! I'll rebase this onto your branch and add both of our buttons to the section.

**Pull request merged**